idx/bigtable/bigtable.go

```diff
@@ -493,10 +496,17 @@ func (b *BigtableIdx) deleteRow(key string) error {
 	return nil
 }
 
-func (b *BigtableIdx) Prune(oldest time.Time) ([]idx.Archive, error) {
-	pre := time.Now()
-	pruned, err := b.MemoryIdx.Prune(oldest)
-	statPruneDuration.Value(time.Since(pre))
+func (b *BigtableIdx) Prune(now time.Time) ([]idx.Archive, error) {
+	log.Info("bigtable-idx: start pruning of series")
```

Comment:
`bigtable-idx: starting prune of series` ?

Reply:
same thing no?